ContestUtil/src/org/icpc/tools/contest/util/ImagesGenerator.java

```diff
@@ -434,8 +437,11 @@ private static File getDefaultFile(File folder, String property) {
 	}
 
 	public void generateOrganizationLogos() {
-		ImageSpec[] spec = new ImageSpec[] { new ImageSpec(DESKTOP, false), new ImageSpec(ICON, true),
-				new ImageSpec(TILE, true) };
+		ImageSpec[] spec = new ImageSpec[] { new ImageSpec(DESKTOP, false), new ImageSpec(TILE, true) };
+		spec = Arrays.copyOf(spec, spec.length + ICONS.length);
+		for (int i = 0; i < ICONS.length; i++) {
+			spec[spec.length - ICONS.length + i] = new ImageSpec(ICONS[i], true);
+		}
```

Comment:
If we can do this more cleanly in Java, I'm all up for it. Copilot suggested this ;)